site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -9806,39 +9810,42 @@ private String transferSiteResource(String oSiteId, String nSiteId, String siteA
 				
 				ref = EntityManager.newReference(contentRef);
 				
-				ContentResource resource = m_contentHostingService.getResource(ref.getId());
-				// the new resource
-				ContentResource nResource = null;
-				String nResourceId = resource.getId().replaceAll(oSiteId, nSiteId);
 				try
 				{
-					nResource = m_contentHostingService.getResource(nResourceId);
-				}
-				catch (Exception n2Exception)
-				{
-					// copy the resource then
+					ContentResource resource = m_contentHostingService.getResource(ref.getId());
```

Comment:
Moving this into the first try/catch means potentially you wouldn't have updated the SiteId?